orient_express/deployment.py

```diff
@@ -0,0 +1,97 @@
+import logging
+import argparse
+from kfp import compiler, dsl
+
+import conf
+import google.cloud.aiplatform as aip
+
+
+def delete_old_schedules(schedule_name):
+    for schedule in aip.PipelineJobSchedule.list(
+        filter=f'display_name="{schedule_name}"', order_by="create_time desc"
+    ):
+        logging.info(f"Deleting the old schedule {schedule}")
+        schedule.delete()
+
+
+def deploy_pipeline(
```

Comment:
can we add type parameters for all the parameters and mark Optional parameters explicitly.

Reply:
good point, added type hints